src/test/java/org/folio/rest/impl/AccountsAPITest.java

```diff
@@ -300,6 +303,62 @@ public void shouldNotAddLoanIdIfItIsNotValidInAccount() {
     assertFalse(eventPayload.containsKey("loanId"));
   }
 
+  @Test
+  public void payCheckAmountShouldBeAllowed() {
```

Comment:
Should we maybe extract these new tests into a separate class? `AccountsActionChecksAPITests`?

Reply:
extracted